README.md

```diff
@@ -116,6 +117,24 @@ Output of `A | B` - the union of A and B (order is not guaranteed)
 
 #
 
+#### Custom Hash Function
+
+To use a custom hash function, you can use the following types:
+
+- `HashSetUnmanagedWithContext`
+- `HashSetManagedWithContext`
+
+Example:
+
+```zig
+    const context = MyContext{};
```

Comment:
Since this part of the README.md is for demonstrating a CustomHashFunction, perhaps it should show a complete example of one would supply the hash/equality funcs.

Reply:
Yes indeed, added in https://github.com/deckarep/ziglang-set/pull/13/commits/a6e4bbfb09d5c252e9d46e149b3d9df8f09134a3